WordPress/Classes/Utility/WPAvatarSource.m

```diff
@@ -129,7 +129,7 @@ - (WPAvatarSourceType)parseURL:(NSURL *)url forAvatarHash:(NSString **)avatarHas
     if ([components count] > 2) {
         NSString *type = components[1];
         NSString *hash = components[2];
-        if ([hash length] == 32) {
+        if ([hash length] == 32 || [hash length] == 64) {
```

Comment:
This makes sense given the hash is now 64 characters 👍 
nitpick: I don't know why this has to verify the hash's length, since it seems unlikely we'll get the wrong length and if we do, I don't think there's harm in it. What will happen AFAIK is that we'll just [request the avatar/blavatar](https://en.gravatar.com/site/implement/images/) and it'll fail.